main/design_documents/aarch64_sve_pcs.md

```diff
@@ -0,0 +1,27 @@
+# Design Document for aarch64_sve_pcs
+
+This patch proposes the introduction of aarch64_sve_pcs.
```

Comment:
This is a design doc, not a commit message :)

```suggestion
```

Reply:
Removed.